4337/contracts/Safe4337Module.sol

```diff
@@ -167,7 +167,7 @@ contract Safe4337Module is IAccount, HandlerContext, CompatibilityFallbackHandle
     }
 
     /**
-     * @dev Validates that the user operation is correctly signed. Reverts if signatures are invalid.
+     * @dev Validates that the user operation is correctly signed and returns an ERC-4337 packed validation data.
```

Comment:
Minor request (non-blocking): Can we reference what ERC-4337 packed validation data is and how it build? (i.e. a link to the relevant ERC section)